pkg/controller/bootstrap/bootstrap.go

```diff
@@ -70,6 +72,7 @@ func (b *Bootstrap) Run(destDir string) error {
 	decoder := codecFactory.UniversalDecoder(mcfgv1.GroupVersion, apioperatorsv1alpha1.GroupVersion, apicfgv1.GroupVersion)
 
 	var cconfig *mcfgv1.ControllerConfig
+	var kconfig *mcfgv1.KubeletConfig
```

Comment:
Shouldn't this be an array as well? We support multiple kubeletconfigs, and there could be 1 per pool as well